vm/src/stdlib/math.rs

```diff
@@ -272,9 +272,55 @@ fn math_ldexp(
     Ok(value * (2_f64).powf(objint::try_float(i.as_bigint(), vm)?))
 }
 
-fn math_gcd(a: PyIntRef, b: PyIntRef) -> BigInt {
+fn math_perf_arb_len_int_op<F>(
+    args: PyFuncArgs,
```

Comment:
You can use here `Args<PyIntRef>`. Look at `PySetInner::update` for example. This will remove some of the argument parsing code.